xline/src/main.rs

```diff
@@ -183,11 +184,11 @@ struct ServerArgs {
     #[clap(long, value_parser = parse_duration, default_value = "50ms")]
     rpc_timeout: Duration,
     /// Candidate election timeout
-    #[clap(long, value_parser = parse_duration, default_value = "1s")]
-    candidate_timeout: Duration,
+    #[clap(long, default_value_t = default_candidate_timeout_round())]
+    candidate_timeout: u8,
```

Comment:
If `candidate_timeout` is not the same type as other timeouts, like `rpc_timeout`, we'd better rename it as `candidate_timeout_tick`, or something like that. Imagine that `xline --rpc-timout=30ms --candidate_timeout=10`, it seems kinda weird.

Reply:
Fixed